packages/lavapack/src/runtime-template.js

```diff
@@ -51,6 +51,19 @@
 
 
   function loadModuleData (moduleId) {
+    // eslint-disable-next-line node/global-require
+    if (typeof window === 'undefined' && require('resolve').isCore(moduleId)) {
```

Comment:
I'm realizing now that this won't work with a standalone bundle since `resolve` is not going to be available. Could we bundle `is-core-module` with the runtime? 🤔 @naugtur

Reply:
An alternative would be to use `node:module`: https://nodejs.org/api/module.html#moduleisbuiltinmodulename

Perhaps a cleaner solution?

Reply:
I've pushed a commit that uses the Node.js API for this, it does require at least Node 16 though

Reply:
I'm not sure how we could support ESM imports with this since dynamic ESM imports are async 🤔